code/game/atoms.dm

```diff
@@ -1,7 +1,7 @@
 /atom
 	layer = TURF_LAYER
 	plane = GAME_PLANE
-	appearance_flags = TILE_BOUND|PIXEL_SCALE
+	appearance_flags = TILE_BOUND|PIXEL_SCALE|LONG_GLIDE
```

Comment:
what does this long glide flag do?

Reply:
`glide_size` controls the speed of gliding animations when gliding between two turfs happens. Problem is that the API is dumb and it's actually represented in pixels/frame, instead of something sane like the time the glide takes.

What this flag does is make diagonal glides take `sqrt(2)` times longer so that glide sizes work implicitly diagonally or cardinally.